index.html

```diff
@@ -20,377 +20,273 @@
+        <div class="logo">
+          <span>Felix</span>
         </div>
-        <div class="name-2"><h3>I’am a Software Developer</h3></div>
-        <div class="p-name">
-            <p>
-                I can help you build a product, 
-                feature or website Look through some of my work and experience! 
-                If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.
-            </p>
+        <div class="menu">
+          <ul>
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+            <li><i class="bi bi-envelope"></i></li>
+          </ul>
         </div>
-
-        <section class="social-illustration">
-
-            <div class="socialmedia">
-                <ul>
-                    <li><a href="#"><img src="assets/Vector.png" alt=""/></a></li>
-                    <li><a href="#"><img src="assets/Group.png" alt=""/></a></li>
-                    <li><a href="#"><img src="assets/Vector2.png" alt=""/></a></li>
-                    <li><a href="#"><img src="assets/Vector3.png" alt=""/></a></li>
-                    <li><a href="#"><img src="assets/Vector4.png" alt=""/></a></li>
-                    
-                </ul>
- 
-    
-                
-    
-    
-            </div>
-
-        </section>     
-    </section>
-    <section class="recent-works">
-      <h1>My Recent Works</h1>
-      <div class="line-rule"></div>
-
-      <div class="img-placeholder">
-        <img src="assets/img-placeholder.png" alt=""/>
       </div>
 
-      <div class="post">
-        <h2>Multi-Post Stories</h2>
-      </div>
-      <div class="content">
-        <p>
-          A daily selection of privately personalized reads; 
-          no accounts or sign-ups required. 
-          has been the industry's standard dummy text ever since the 1500s, 
-          when an unknown printer took a standard dummy text.
-        </p>
-      </div>
-
-      <div class="tag">
+      <section class="headline">
+        <h1>Hey There.<br>I’m Felix</h1>
+        <span>I’am a Software Developer</span>
+        <p>I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+      </section>
+      <div class="socials">
         <ul>
-          <li>Css</li>
-          <li>html</li>
-          <li>bootstrap</li>
-          <li>Ruby</li>
+          <li><img src="assets/Vector.png" alt=""/></li>
+          <li><img src="assets/Group.png" alt=""/></li>
+          <li><img src="assets/Vector2.png" alt=""/></li>
+          <li><img src="assets/Vector3.png" alt=""/></li>
+          <li><img src="assets/Vector4.png" alt=""/></li>
         </ul>
       </div>
+    </section>
+    <section class="recent-works">
+      <div class="recent-works-title">
+        <h1>My Recent Works</h1>
```

Comment:
- [ ] [Optional] I would suggest you please use `hx` tag here like `h2, h3,h4` because only we use the `h1` tag per page and it represents the main heading of the page.